.gitignore

```diff
@@ -55,6 +55,7 @@ reuters/
 benchmarks/bench_covertype_data/
 
 *.prefs
+.project
```

Comment:
Usually we don't add unrelated change. If you could revert this one.

Reply:
Done in 4c5f8d2